### ISSUE TITLE
Enhancement[Do / Don't component]: Icon adornments need to be white (not knocked out)

### ISSUE THREAD

**Issue by jeanservaas** [carbon-design-system/carbon-website#1272]
## Enhancement[Do / Don't component]: Icon adornments need to be white (not knocked out)

The check mark and the 'x' adornment on the Do / Don't component in Carbon are knocked out of the icons... They need to be white, like they are on the DL site in order to avoid accessibility issues on mid-tone backgrounds.

@chrisconnors-ibm just solved this issue for the warning notification icon; we're going to have to do the same thing here by creating new assets for these icons that have layered paths.

### Where

- video do / don't components (motion page)

- do / don't components (all other elements pages)

![image.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/290697ae-7d3f-460c-bee6-aa7bd7e0edfb)

**Comment by chrisconnors-ibm** [comment]
once https://github.com/IBM/carbon-elements/pull/399 is approved/merged, all 8 `/*/misuse.svg` and `/*/checkmark--filled.svg` fill have the adornment duplicated in a path labeled `inner-path`. style opacity and fill of `inner-path` independently from the bg path.

**Comment by chrisconnors-ibm** [comment]
is there a separate issue for dev to style the `inner-path` for these uses?

**Comment by chrisconnors-ibm** [comment]
@alisonjoseph we're going to here from - _people_ - about this @ release

**Comment by alisonjoseph** [comment]
need to add this update to the addons/gatsby-theme

**Comment by jeanservaas** [comment]
Not stale, we still need this done for accessibility, it's going to be an issue in the gatsby template too @vpicone @alisonjoseph
